hail/python/hail/experimental/ldscsim.py

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Feb 15 08:25:59 2019
+
+ldsc simulation framework
+
+@author: nbaya
+"""
+
```

Comment:
You'll need to add something to hl.experimental.__init__.py to import and expose functions you want to be available at that level.

Probably just 'simulate' which we should rename 'simulate_phenotypes' to be clear at that level.

It also is probably worth writing a full docstring for all the parameters, and providing some notes on what it does. But you can do that in a separate PR if you want.

Reply:
There are a couple of functions related to variant annotations and the annotation-informed model which I would also like to make available. Should I import them from this module or should I create a new module just for these methods?

Reply:
A python file behaves like a module -- You can do `hl.experimental.ldscsim.foo` for instance. You should put at most one or two functions in the upper `experimental` namespace, but you can have whatever you want here.

Reply:
okay, great